### PR TITLE
Update tocalls.yaml

### PR DESCRIPTION
**Pull request by F5OPV**
Hi Hessu,

would it be possible to add a new class to the tocalls list: AoT class, relative to APRS of Things, mcu-driven devices for beaconing, telemetry, remote controlers, reporters... objects?
The aprs equivalent of IoT but specifically for ham radio operators dealing with making of mcu ham stuff.

Any project about creation of new 'tech' icons in the aprs symbols list? (missing here icons with all my APRS objects hi !)
As coloured symbols 1 to 9 are not used in the alternate symbols table, it would be fine to get possibility of using color icon of primary table with alternate letters A-Z giving by this way numerous different new symbols.

Thanks in advance for answer Hessu.

PS Request of a new class of telemetry devices: APSFTL 

73 de F5OPV Pierre

### CODE REVIEW

**Comment by hessu** [comment]
Sorry for the slow reply. In principle, maybe, but most simple trackers and old telemetry devices are MCU devices (think of the TinyTrak4, or Tracker2, or any of those), and rebranding them as "AoT" would be weird. They're still trackers and if we move them to "AoT" people will be confused. So maybe some further thinking about the scope of this class is needed.

The primary document for tocall allocations is still http://aprs.org/aprs11/tocalls.txt so please email Bob Bruninga asking to allocate the callsign before we can add it here. Thanks.

Technical changes required for a PR:
* Use a descriptive commit message and PR title, as seen in previous commit logs. The message should say what was changed: "+ APMI06, Microsat WX3in1 Plus 2.0" for example
* lower-case identifier for the "aot" class, it will not be shown to end users, it's just an identifier in the database

**Comment by F5OPV** [comment]
Hi Hessu,
thanks for reply.

First sorry but my English is not very 'rich'.

Ok for AoT, it was just a suggest, as everybody's talking today abt IoT based MCU, and I was thinking it would be a good idea to add it as a new general class as it would be in my humble advice to add 'Bot'' (Robot or Automate in french).

I am not very familiar with github, so sorry if there are errors in my commit request about APSFTL.
Bob allocated me 3 or 4 years ago APSF??, so I'd like to add Telemetry devices I build by using a specificTL suffix to APSF: is it necessary to ask specific allocation to Bob before asking you, as sprcific purpose is to display the type of device on the info page of aprs.fi and parsing of destination call on APRS-IS ?
I already have built wx stations (APSFWX), APRS/LoRA Gateways (APSFLG) you allocated me previously, but I didn't aks this allocation to Bob as I was thinking subdevices of APSF?? could be allocated freely .

About my previous commit APSFTL, thankyou for telling me if there was a technical error, if I can do a new request to you according to the **necessary technical changes** or if I must **request it first to Bob**.
For future, I would like to add new sub-callsigns (mcu repeaters APSFRP, propagation info reporters APSFPI, crossband gateways APSFGW...), to the APSF?? main callsign allocated to me by Bob,  so your answer would be of great interest for me.

Thanks in advance, Hessu and
73 de F5OPV

File: tocalls.yaml [hunk 1]
```diff
@@ -862,6 +866,12 @@ tocalls:
    class: digi
    os: embedded
 
+ - tocall: APSFTL
+   vendor: F5OPV, SFCP_LABS
+   model: LoRa/APRS Telemetry Reporter
+   class: AoT
+   os: embedded
```
Comment (NoseyNick) [inline tocalls.yaml:873]: APSFTL not seen in http://aprs.org/aprs11/tocalls.txt - you should probably get Bob to add it there first?

**Comment by hessu** [comment]
Thanks, merged the other PR after editing commit message a bit.

**Pull request closed without merging**